src/platform/forms-system/src/js/fields/ArrayField.jsx

```diff
@@ -402,6 +421,67 @@ export default class ArrayField extends React.Component {
             const useCardStyling = notLastOrMultipleRows;
             const CardOrDiv = useVaCards && useCardStyling ? VaCard : 'div';
 
+            const displayFields = confirmRemoveDisplayFields
+              .map(dataPath => {
+                const keys = dataPath.split('.');
+                let value = formData?.[index];
+                keys.forEach(key => {
+                  value = value ? value[key] : undefined;
+                });
+
+                if (dataPath === 'serviceBranch') {
```

Comment:
we don't want implementation specific code in forms-system. this would need to be abstracted